.github/workflows/backport.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     name: Backport
     steps:
       - name: Backport
-        uses: zephyrproject-rtos/action-backport@v1.1.99
+        uses: zephyrproject-rtos/action-backport@v1.1.1-1
```

Comment:
this looks like a downgrade? :)

Reply:
hehe I changed the numbering scheme because upstream went from 1.1.0 to 1.1.1.
I initially had 1.1.99 but that wasn't gonna work so I adopted the Debian guidelines instead, using 1.1.1-1.